lib/alchemy/config_loader.rb

```diff
@@ -0,0 +1,69 @@
+module Alchemy
+  class ConfigLoader
+    attr_reader :which
+
+    def initialize(which)
+      @which = which.sub(/\.yml$/, '')
+    end
+
+    # Reads the config file named +<which>.yml+ from +config/alchemy/+ folder.
+    #
+    def load_all
+      if paths.empty?
+        raise LoadError, "Could not find #{which}.yml file! Please run `rails generate alchemy:scaffold`"
+      else
+        # OPTIMIZE: remove duplicates?
+        paths.map(&method(:load_file)).reduce(&method(:merge))
+      end
+    end
+
+    def file_name
```

Comment:
You only need to test the `load_all` method, since you only want to test the behaviour not the implementation.